### ISSUE TITLE
[Market] NPE getRoomInfo()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#160]
```
java.lang.IllegalStateException: java.lang.NullPointerException
at com.googlecode.gtalksms.XmppManager.initConnection(XmppManager.java:519)
at com.googlecode.gtalksms.XmppManager.start(XmppManager.java:150)
at 
com.googlecode.gtalksms.XmppManager.xmppRequestStateChange(XmppManager.java:215)
at com.googlecode.gtalksms.MainService.onHandleIntent(MainService.java:178)
at 
com.googlecode.gtalksms.MainService$ServiceHandler.handleMessage(MainService.jav
a:122)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:143)
at android.os.HandlerThread.run(HandlerThread.java:60)
Caused by: java.lang.NullPointerException
at org.jivesoftware.smackx.muc.RoomInfo.<init>(RoomInfo.java:91)
at org.jivesoftware.smackx.muc.MultiUserChat.getRoomInfo(MultiUserChat.java:238)
at com.googlecode.gtalksms.xmpp.XmppMuc.getRoomInfo(XmppMuc.java:333)
at com.googlecode.gtalksms.xmpp.XmppMuc.rejoinRooms(XmppMuc.java:232)
at com.googlecode.gtalksms.xmpp.XmppMuc.access$3(XmppMuc.java:226)
at com.googlecode.gtalksms.xmpp.XmppMuc$1.newConnection(XmppMuc.java:63)
at com.googlecode.gtalksms.XmppManager.informListeners(XmppManager.java:546)
at com.googlecode.gtalksms.XmppManager.initConnection(XmppManager.java:499)

V2.2.2
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 6 May 2011 at 7:11

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed with smack 3.2.0: 
http://fisheye.igniterealtime.org/browse/smack/trunk/source/org/jivesoftware/sma
ckx/muc/RoomInfo.java?r2=12200&r1=7071
```

Original comment by `fschm...@gmail.com` on 21 May 2011 at 3:48
- Changed title: **[Market] NPE getRoomInfo()**
- Changed state: **WaitingForRelease**
- Added labels: **Priority-Medium**
- Removed labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.2.4
```

Original comment by `Florent....@gmail.com` on 13 Jun 2011 at 1:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
